### PR TITLE
Add support for standard save keyboard shortcut

### LINKED ISSUE #304: Support standard keyboard shortcut to save
### Description:

Many online WYSIWYG editors such as Google Docs, Gmail, etc. hook into the standard OS save shortcut (`ctrl`+`S` on Windows, `cmd`+`S` on Mac, etc) to prevent the default browser behavior of trying to save the current page content as an html file.

This should be implemented in Jekyll admin. The current default behavior is jarring, especially because many users will be in the habit of mashing ctrl+s frequently, only to have to close the default dialogue and manually click the save button instead.

### Tell us a bit about yourself:

- jekyll-admin v0.3.0
- jekyll v3.3.1
- tested on Windows 10 and MacOS 10.10
- Chrome v56.0.2924.87 (64-bit)

### PR DESCRIPTION
**Pull request by mikefowler**
## Overview

Adds support for the standard "Save" keyboard shortcut: Command+s on OSX,
or Ctrl+s on Linux and Windows.

This change introduces a new dependency, react-hotkeys, which will make
it trivial to add additional keyboard shortcuts in the future.

Closes #304

cc: @JohannesMP @mertkahyaoglu

### CODE REVIEW

**Comment by mertkahyaoglu** [comment]
@mikefowler This is really great 👍  

Could you add the save shortcut for `Configuration` too?

File: src/containers/App.js [hunk 1]
```diff
@@ -9,7 +12,9 @@ class App extends Component {
 
   render() {
     return (
-      <div className="wrapper">
+      <HotKeys
```
Comment (mertkahyaoglu) [inline src/containers/App.js]: Do we need `HotKeys` here?

Comment (mikefowler) [inline src/containers/App.js]: Yes. As far as I can tell, `react-hotkeys` requires you to wrap the entire React tree that will be listening for hotkeys.

**Comment by mikefowler** [comment]
@mertkahyaoglu, I added the shortcut to `Configuration` as well, and added a helper for preventing default events 👍

**Comment by mertkahyaoglu** [comment]
Thank you @mikefowler , very useful addition 👍 

Great to have you here as a first-time contributor ❤️

**Comment by mikefowler** [comment]
@mertkahyaoglu <3

**Pull request merged**